packages/core/src/textures/resources/ImageBitmapResource.ts

```diff
@@ -1,27 +1,203 @@
+import { ALPHA_MODES } from '@pixi/constants';
+import { settings } from '@pixi/settings';
```

Comment:
```suggestion
import { ALPHA_MODES, settings } from '@pixi/core';
```

Reply:
`ImageResource` and `ImageBitmapResource` are inside of `@pixi/core`, they can't import `@pixi/core` itself.

Reply:
Yeah, my bad!